src/orm/firebase.ts

```diff
@@ -32,6 +32,8 @@ if (__DEVELOPMENT__) {
     })
 }
 
+db.enablePersistence({ synchronizeTabs: true })
```

Comment:
I looked at the docs, and still don't quite understand what it means to synchronize all open tabs. Is that across all users of the application that have a tab open or a single user to enable the db to persist the tab they currently have open. 

Here is what im referencing: https://firebase.google.com/docs/reference/js/firebase.firestore.PersistenceSettings

Reply:
As far as I know, they use some sort of shared worker/service worker in firebase to do some of the caching and if you don't have `synchronizeTabs` enabled it will only work in 1 tab at a time. With it on you can have the multiple tabs open at the same time working in all of them